src/downloadmanager.cpp

```diff
@@ -140,7 +140,8 @@ bool DownloadManager::moveMyAppPackage(QString path)
         uid_t uid = getuid();
         // assumes that correct groupname is same as username (e.g. nemo:nemo)
         int gid = getgrnam(getpwuid(uid)->pw_name)->gr_gid;
-        chown(aptoideDownloadPath.toLatin1().data(), uid, gid);
+        int success = chown(aptoideDownloadPath.toLatin1().data(), uid, gid);
+        Q_UNUSED(success);
```

Comment:
How is that possible? Casting function return value to void is one of the most common methods of silencing unused return value compiler warnings in C/C++.